### PR TITLE
APPDRUSTI-4

### PR DESCRIPTION
**Pull request by BOSSHK-SPEC**

### CODE REVIEW

File: lib/LoginAndRegistration.dart [hunk 1]
```diff
@@ -19,12 +22,20 @@ class LoginAndRegisterationPage  extends StatefulWidget{
 }
 
 class LoginAndRegisterationPageState extends State{
+  int _value = 1;
```
Comment (sahana-drusti) [inline lib/LoginAndRegistration.dart]: if it is not used, please remove it

File: lib/LoginAndRegistration.dart [hunk 2]
```diff
@@ -19,12 +22,20 @@ class LoginAndRegisterationPage  extends StatefulWidget{
...
   bool login = true;
   bool signup = false;
   List<bool> isSelect = [true,false];
   final _formKey = GlobalKey<FormState>();
+  List state=["karnataka","goa","mp"];
+  var stateValue="karnataka";
+  List district=["hassan","banglore"];
+  var districtValue="hassan";
+  List thalluk=["alur","belur"];
```
Comment (sahana-drusti) [inline lib/LoginAndRegistration.dart]: taluk spelling

File: lib/LoginAndRegistration.dart [hunk 3]
```diff
@@ -33,7 +44,9 @@ class LoginAndRegisterationPageState extends State{
           body:Column(
             children: [
               ToggleButtons(
+
```
Comment (sahana-drusti) [inline lib/LoginAndRegistration.dart]: format the code

File: lib/LoginAndRegistration.dart [hunk 4]
```diff
@@ -69,46 +82,73 @@ class LoginAndRegisterationPageState extends State{
                     mainAxisAlignment: MainAxisAlignment.center,
                     children: [
                       TextFormField(
+                        decoration: new InputDecoration(
+                       focusedBorder: OutlineInputBorder(
+                       borderSide: BorderSide(color: Colors.black, width: 2.0),
+                       ),
+                        enabledBorder: OutlineInputBorder(
+                        borderSide: BorderSide(color: Colors.black, width: 2.0),
```
Comment (sahana-drusti) [inline lib/LoginAndRegistration.dart]: change width to 1

File: lib/LoginAndRegistration.dart [hunk 5]
```diff
@@ -69,46 +82,73 @@ class LoginAndRegisterationPageState extends State{
...
+                      ),
+                          hintText:'username'
```
Comment (sahana-drusti) [inline lib/LoginAndRegistration.dart]: use UserName

File: lib/LoginAndRegistration.dart [hunk 6]
```diff
@@ -142,112 +182,211 @@ class LoginAndRegisterationPageState extends State{
...
                       TextFormField(
-                        validator: (val){
-                          if(val == null || val.isEmpty){
-                            return 'Please enter Password';
-                          }
-                        },
-                        decoration: InputDecoration(
-                            hintText: 'Enter Reg Number'
+
+                          decoration:new InputDecoration(
```
Comment (sahana-drusti) [inline lib/LoginAndRegistration.dart]: add email validation

File: lib/LoginAndRegistration.dart [hunk 7]
```diff
@@ -142,112 +182,211 @@ class LoginAndRegisterationPageState extends State{
...
-                            return 'Please enter Password';
-                          }
-                        },
-                        decoration: InputDecoration(
-                            hintText: 'Enter Reg Number'
+                      DropdownButtonFormField(
+                        decoration:new InputDecoration(
+                          focusedErrorBorder: OutlineInputBorder(
+                            borderSide: BorderSide(color: Colors.black, width: 2.0),
+                          ),
```
Comment (sahana-drusti) [inline lib/LoginAndRegistration.dart]: change width to 1 everywhere

File: lib/LoginAndRegistration.dart (hunk 5 above)
Comment (BOSSHK-SPEC) [reply lib/LoginAndRegistration.dart]: ok

File: lib/LoginAndRegistration.dart (hunk 6 above)
Comment (BOSSHK-SPEC) [reply lib/LoginAndRegistration.dart]: ok

File: lib/LoginAndRegistration.dart (hunk 7 above)
Comment (BOSSHK-SPEC) [reply lib/LoginAndRegistration.dart]: ok

**Review by sahana-drusti — APPROVED** [review]

**Pull request merged**